pyproject.toml

```diff
@@ -26,7 +26,9 @@ dependencies = [
     "slack_sdk>=3.35.0",
     "nest-asyncio>=1.6.0",
     "pydantic-ai[logfire]>=0.1.8",
-    "opik>=1.7.22"
+    "opik>=1.7.22",
+    "yfinance>=0.2.63",
+    "qiskit-ibm-catalog>=0.7.0",
```

Comment:
would it be worth adding dependencies for the demos in their own [dependency group](https://docs.astral.sh/uv/concepts/projects/dependencies/#dependency-groups)?